packages/query-async-storage-persister/package.json

```diff
@@ -10,10 +10,17 @@
     "type": "github",
     "url": "https://github.com/sponsors/tannerlinsley"
   },
-  "module": "build/esm/index.js",
-  "main": "build/cjs/index.js",
+  "module": "build/lib/index.mjs",
+  "main": "build/lib/index.js",
   "browser": "build/umd/index.production.js",
   "types": "build/types/packages/query-async-storage-persister/src/index.d.ts",
+  "exports": {
+    ".": {
+      "import": "./build/lib/index.mjs",
+      "default": "./build/lib/index.js"
+    },
+    "./package.json": "./package.json"
+  },
   "files": [
     "build/*",
```

Comment:
the build directory contains things that we probably shouldn't ship, like `stats.html` etc

Reply:
Picked more granular paths. But this could still change if i find a way to bundle types into lib properly